src/MvvmNano.Forms/NanoMasterDetailPage.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Linq; 
+using Xamarin.Forms;
+
+namespace MvvmNano.Forms
+{
+    /// <summary>
+    /// A MasterDetailPage implementation that fits to the MvvmNano framework.
+    /// </summary>
+    public class NanoMasterDetailPage :  MasterDetailPage
```

Comment:
This is basically the same as `MvvmNanoMasterDetailPage<TViewModel>`, right? If that's correct, I would like those two being merged into a single _MvvmNanoMasterDetailPage.cs_ and this one re-named to `MvvmNanoMasterDetailPage` (without <T>). So we don't have an exception in naming things.